### PR TITLE
ALY VSS with one country

### PR DESCRIPTION
**Pull request by SiddarGu**
This solves an issue that ALY/VSS cannot take a single country as its argument.
Solves #52

### CODE REVIEW

**Comment by mjspeck** [comment]
Looks like some tests may need to be fixed.

File: src/daidepp/grammar/grammar.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "try": '"TRY" lpar try_tokens (ws try_tokens)* rpar',
     "huh": '"HUH" lpar press_message rpar',
     "prp": '"PRP" lpar arrangement rpar',
-    "aly_vss": '"ALY" lpar power (ws power)+ rpar "VSS" lpar power (ws power)+ rpar',
+    "aly_vss": '"ALY" lpar power (ws power)* rpar "VSS" lpar power (ws power)* rpar',
```
Comment (mjspeck) [inline src/daidepp/grammar/grammar.py:54]: This should actually be `"aly_vss": '"ALY" lpar power (ws power)+ rpar "VSS" lpar power (ws power)* rpar',`. A single power cannot enter an alliance without another power.

**Review by mjspeck — CHANGES_REQUESTED** [review]

File: src/daidepp/grammar/grammar.py (hunk 1 above)
Comment (mjspeck) [inline src/daidepp/grammar/grammar.py:54]:
```suggestion
    "aly_vss": '"ALY" lpar power (ws power)+ rpar "VSS" lpar power (ws power)* rpar',
```
A single power cannot enter an alliance without another power.

**Review by mjspeck — CHANGES_REQUESTED** [review]
Cynnovative can update the test cases to pass once the necessary change is added.

**Comment by SiddarGu** [comment]
> Since we don't have push access to the fork, you'll need to make this update in `tests/keywords/test_press_keywords.py`:
> 
> ```python
> @pytest.mark.parametrize(
>     # fmt: off
>     ["input"],
>     [
>         ((["AUS"], ["ENG"],),),
>         ((["FRA"], ["GER", "TUR"],),),
>         ((["AUS", "FRA"],[]),),
>     ],
>     # fmt: on
> )
> def test_ALYVSS_bad(input):
>     with pytest.raises(Exception):
>         ALYVSS(*input)
> ```

grammar and test are updated.

**Pull request closed without merging**